packages/generators/commons/src/BaseGeneratorConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from "zod";
+
+// Common per-generator configuration flags
+export interface BaseGeneratorConfig {
+    clientClassName: string;
+    noOptionalProperties: boolean;
```

Comment:
what is this?

Reply:
This was pulled from the ts schema thinking it was a universal config, removed!